src/lib/stores/theme.store.ts

```diff
@@ -1,11 +1,17 @@
 import type { Theme } from "$lib/types/theme";
 import { applyTheme, initTheme } from "$lib/utils/theme.utils";
-import { writable } from "svelte/store";
+import { writable, type Readable } from "svelte/store";
+
+export type ThemeStoreData = Theme | undefined;
+
+export interface ThemeStore extends Readable<ThemeStoreData> {
+  select: (theme: Theme) => void;
+}
 
 const initialTheme: Theme | undefined = initTheme();
```

Comment:
[nitpick]
```suggestion
const initialTheme: ThemeStoreData = initTheme();
```

Reply:
I developed the interface in another PR and missed copying this line over. Good catch, thanks!